content/tutorial/tutorial.md

```diff
@@ -448,19 +454,19 @@ class Square extends React.Component {
 }
 ```
 
-When a Square is clicked, the `onClick` function provided by the Board is called. Here's a review of how this is achieved:
+जब एक स्क्वायर पर क्लिक किया जाता है, तो Board द्वारा प्रदान किया गया `onClick` फ़ंक्शन कॉल होता है। यहां बताया गया है कि यह कैसे हासिल किया जाता है:
 
-1. The `onClick` prop on the built-in DOM `<button>` component tells React to set up a click event listener.
-2. When the button is clicked, React will call the `onClick` event handler that is defined in Square's `render()` method.
-3. This event handler calls `this.props.onClick()`. The Square's `onClick` prop was specified by the Board.
-4. Since the Board passed `onClick={() => this.handleClick(i)}` to Square, the Square calls `this.handleClick(i)` when clicked.
-5. We have not defined the `handleClick()` method yet, so our code crashes. If you click a square now, you should see a red error screen saying something like "this.handleClick is not a function".
+1. अंतर्निहित 'DOM' पर `onClick` prop `<button>` कौम्पोनॅन्ट क्लिक इवेंट लिस्टनेर स्थापित करने के लिए React को बताता है।
+2. जब बटन पर क्लिक किया जाता है, तो React `onClick` ईवेंट हैंडलर को कॉल करेगा जिसे Square के `render()` मेथड में परिभाषित किया गया है।
+3. यह ईवेंट हैंडलर `this.props.onClick()` को कॉल करता है। Board द्वारा Square के `onClick` prop को निर्दिष्ट किया गया था।
+4. चूँकि Board ने `onClick = {() => this.handleClick (i)}` को Square में दिया, Square ने क्लिक करने पर `this.handleClick(i)` को कॉल किया।
+5. हमने अभी तक `handleClick()` मेथड को परिभाषित नहीं किया है, इसलिए हमारा कोड क्रैश हो गया है। यदि आप अभी एक वर्ग क्लिक करते हैं, तो आपको एक लाल एरर स्क्रीन दिखनी चाहिए, जिसमें ये एरर दिखेगा "this.handleClick कोई फ़ंक्शन नहीं है"।
```

Comment:
"this.handleClick is not a function" is a literal error, should we translate it? Or put the translation next to brackets next to the original error?

Reply:
on reading this now, I think we should keep it in english without any translation, as exact same english text will be appearing during error